run-checks

```diff
@@ -221,24 +221,28 @@ if [ "$STATIC" = 1 ]; then
         ./tests/lib/external/snapd-testing-tools/utils/spread-shellcheck spread.yaml tests --exclude "$exclude_tools_path"
     fi
 
-    echo "Checking spelling errors"
-    if ! command -v misspell >/dev/null; then
-        goinstall github.com/client9/misspell/cmd/misspell
+    if [ -z "${SKIP_MISSPELL-}" ]; then
+        echo "Checking spelling errors"
+        if ! command -v misspell >/dev/null; then
+            goinstall github.com/client9/misspell/cmd/misspell
+        fi
+        # FIXME: auter is only misspelled in the changelog so we should fix there
+        # PROCES is used in the seccomp tests (PRIO_PROCES{,S,SS})
+        # exportfs is used in the nfs-support test
+        MISSPELL_IGNORE="auther,PROCES,PROCESSS,proces,processs,exportfs"
+        git ls-files -z -- . ':!:./po' ':!:./vendor' ':!:./c-vendor' ':!:./cmd/libsnap-confine-private/bpf/vendor' |
+            xargs -0 misspell -error -i "$MISSPELL_IGNORE"
     fi
-    # FIXME: auter is only misspelled in the changelog so we should fix there
-    # PROCES is used in the seccomp tests (PRIO_PROCES{,S,SS})
-    # exportfs is used in the nfs-support test
-    MISSPELL_IGNORE="auther,PROCES,PROCESSS,proces,processs,exportfs"
-    git ls-files -z -- . ':!:./po' ':!:./vendor' ':!:./c-vendor' ':!:./cmd/libsnap-confine-private/bpf/vendor' |
-        xargs -0 misspell -error -i "$MISSPELL_IGNORE"
-
-    if dpkg --compare-versions "$(go version | awk '$3 ~ /^go[0-9]/ {print substr($3, 3)}')" ge 1.12; then
-        echo "Checking for ineffective assignments"
-        if ! command -v ineffassign >/dev/null; then
-            goinstall github.com/gordonklaus/ineffassign
+
+    if [ -z "${SKIP_INEFFASSIGN-}" ]; then
+        if dpkg --compare-versions "$(go version | awk '$3 ~ /^go[0-9]/ {print substr($3, 3)}')" ge 1.12; then
```

Comment:
could we have just bumped 1.12 -> 1.13  here? or do we think this tool will start working again on 1.13 at some point?